apps/src/aichat/views/presentation/ModelCardRow.tsx

```diff
@@ -7,14 +7,16 @@ import moduleStyles from './model-card-row.module.scss';
 
 interface ModelCardRowProps {
   title: string;
-  titleIcon: string;
+  titleIcon?: string;
   expandedContent: string | string[];
+  tooltipText?: string;
 }
 
 const ModelCardRow: React.FunctionComponent<ModelCardRowProps> = ({
   title,
   titleIcon,
   expandedContent,
+  tooltipText, // TODO: Use tooltip text
```

Comment:
Planning on following up with a separate change to display the tooltips in the Presentation view. This is because we'll need to do a bit of refactoring with the `CollapsibleSection` to accommodate the tooltip info icon.